test/__mocks__/text.mock.ts

```diff
@@ -100,6 +100,9 @@ export const SAVE_AND_CONTINUE_BUTTON_TEXT = "Save and continue";
 export const SECOND_NATIONALITY = "Second nationality (optional)";
 export const SECOND_NATIONALITY_HINT = "Dual citizenship (also known as dual nationality) is allowed in some countries.";
 export const PRINT_BUTTON_TEXT = "Print this page";
+export const PUBLIC_REGISTER_NAME_LABEL = "Name of register";
+export const PUBLIC_REGISTER_JURISDICTION_LABEL = "Jurisdiction";
+export const REGISTRATION_NUMBER_LABEL = "Entity&#39;s registration number";
```

Comment:
for the apostrophe, Sam had a similar issue where he had to use the html code, but he could fix it by copying the existing apostrophe from DUE_DILIGENCE_NAME_TEXT on line 87, that seems to be a different character that you can use instead of `&#39;`

Reply:
Replicated this fix.